hooks/persistence-elastic/hook.js

```diff
@@ -3,13 +3,17 @@ const { Client } = require("@elastic/elasticsearch");
 const flatMap = require("lodash.flatmap");
 const chunk = require("lodash.chunk");
 
+const moment = require('moment');
```

Comment:
Might be best to use a different lib than moment as it is kinda deprecated by now: https://momentjs.com/docs/#/-project-status/